blackbox/docs/appendices/release-notes/unreleased.rst

```diff
@@ -21,8 +21,13 @@ Changes
 Fixes
 =====
 
+<<<<<<< HEAD
```

Comment:
this should be dropped manually?

Reply:
It can't auto-fix merge conflicts - they've to be resolved manually and I missed this conflict as I only checked the compile errors. Fixed it now